IPython/utils/process.py

```diff
@@ -41,6 +41,9 @@ def find_cmd(cmd):
         from IPython.utils.process import pycmd2argv
         argv = pycmd2argv(get_ipython_module_path('IPython.terminal.ipapp'))
 
+    Note, The code for pycmd2argv has been removed now as it was not used
+    anywhere. get_ipython_module_path should give us that same result.
```

Comment:
That would be an accurate replacement of what the example currently does.

However, since we wrote this example, Python has added the `-m` flag, which provides an easier way to do what the example is trying to provide. And because of changes to our code, I don't think the original example actually works as intended any more. So my suggestion changes what the example code actually does, but fixes it for what it's intended to do.